javaHapiValidatorLambda/src/test/java/software/amazon/fwoa/ValidatorTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.fwoa;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.common.collect.ImmutableList;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ValidatorTest {
```

Comment:
Do we need a test to validate that we can correctly load an IG and validate a resource with that IG?

Reply:
That will be covered by integ tests.

The thing is that the IGs are added to `src/main/resources/implementationGuides` as if they were part of the lambda source code and it is very tricky to mock those files in a way that makes sense.